addons/langpacks.rst

```diff
@@ -62,12 +62,17 @@ Langpacks: How To…
 How to handle New Languages for Release
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Context: the underlying issue for this lies within `issue-15353`_ and is on the roadmap to be fixed by the AMO team.
-Some related work is happening in H2 2021.
-
+Context: the underlying issue for this lies within `issue-15353`_ and can be
+fixed in addonscript by switching to the v5 AMO API for submission.  In the
+mean time:
+
+-  The channel (listed/unlisted) parameter on submission of new addons is
+   ignored by the v4 API, so new langpacks end up unlisted.
+-  Following releases will attempt to update these langpacks and set them to
+   listed, which will fail.
```

Comment:
You can even add something like: 
```suggestion
   listed, which will fail because required metadata for a listed add-on (categories, license) is not set (and can't be set with the old API).
```